skorch/net.py

```diff
@@ -607,10 +608,14 @@ def fit_loop(self, X, y=None, epochs=None, **fit_params):
             * a dictionary of the former three
             * a list/tuple of the former three
             * a Dataset
+            * a TrainValidDataset
```

Comment:
maybe add: a skorch.dataset.TrainValidDataset, since most people won't know where this comes from.